admin_run_integration_test.go

```diff
@@ -244,6 +252,15 @@ func TestAdminRuns_AdminRunsListOptions_valid(t *testing.T) {
 		err := opts.valid()
 		assert.Error(t, err)
 	})
+
+	t.Run("has trailing comma and trailing space", func(t *testing.T) {
```

Comment:
This test and all the ones for `TestAdminRuns_AdminRunsListOptions_valid` don't really need to be part of integration test suite. We are just testing helper functions and it'd be nice if external contributors could at least have access to running these type of tests. I'd create a unit test file, like I did with `tfe_test.go`, but then suddenly we end up with 2 new unit test files, suddenly it feels like they _should_ have their own directory to live..., suddenly we have the urge to reorganize other files. But this is not the time to create new files or reorganize files.
The subject of reorganizing this project for this upcoming has been discussed quite a bit and we decided to postpone. It would involve too many breaking changes on top of the ones we are already bringing. But I really hope that we can find a solution to this organization/creation of new files for this project. It is not scalable as it is :(